identifiers/country-ee/top_level.csv

```diff
@@ -1,2 +1,14 @@
 id,name
 ocd-division/country:ee,Estonia
+ocd-division/country:ee/ed:1,Haabersti and Põhja-Tallinn and Kristiine (districts of Tallinn)
+ocd-division/country:ee/ed:2,Kesklinn and Lasnamäe and Pirita (districts of Tallinn)
+ocd-division/country:ee/ed:3,Mustamäe and Nõmme (districts of Tallinn)
+ocd-division/country:ee/ed:4,Harju- (excl. Tallinn) and Raplamaa
+ocd-division/country:ee/ed:5,Hiiu- and Lääne- and Saaremaa

```

Comment:
Is there any significance of the trailing hyphen on some of these names? It looks like it should be removed. I see them on the wikipedia page but https://www.riigiteataja.ee/en/eli/ee/514112013015/consolide/current doesn't have them.

Reply:
No trailing hyphen anymore but the same issue.